From 56514bac9745ec263b84adc0479c0c0314902ab9 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Mon, 13 Apr 2015 14:24:38 +0800
Subject: [PATCH] MLK-10646: ARM: dts: Add independent dts for hdmi audio and
 wm8960 audio

wm8960, hdmi can't be used together for they use same SAI interface.
Which is hardware limitation.

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
---
 arch/arm/boot/dts/imx7d-sdb-hdmi-audio.dts | 20 ++++++++++++++++++++
 arch/arm/boot/dts/imx7d-sdb-wm8960.dts     | 20 ++++++++++++++++++++
 2 files changed, 40 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb-hdmi-audio.dts b/arch/arm/boot/dts/imx7d-sdb-hdmi-audio.dts
new file mode 100644
index 0000000..221fdd0
--- /dev/null
+++ b/arch/arm/boot/dts/imx7d-sdb-hdmi-audio.dts
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2015 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx7d-sdb.dts"
+
+/ {
+	sound {
+		status = "disabled";
+	};
+
+	sound-hdmi {
+		status = "okay";
+	};
+};
+
diff --git a/arch/arm/boot/dts/imx7d-sdb-wm8960.dts b/arch/arm/boot/dts/imx7d-sdb-wm8960.dts
new file mode 100644
index 0000000..d9f3829
--- /dev/null
+++ b/arch/arm/boot/dts/imx7d-sdb-wm8960.dts
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2015 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx7d-sdb.dts"
+
+/ {
+	sound {
+		status = "okay";
+	};
+
+	sound-hdmi {
+		status = "disabled";
+	};
+};
+
-- 
1.8.0

